packages/trader/src/Stores/Modules/Trading/trade-store.ts

```diff
@@ -191,6 +191,7 @@ export default class TradeStore extends BaseStore {
     is_market_closed = false;
     previous_symbol = '';
     active_symbols: ActiveSymbols = [];
+    active_symbols_v2: ActiveSymbols = [];
```

Comment:
i’ll replace this with a boolean observable `has_symbols_for_v2` first thing tomorrow since it’s only used to check whether or not we have requested active symbols from v2 in useActiveSymbols.

Reply:
done)